x-pack/plugins/observability_solution/logs_shared/public/components/logging/log_text_stream/log_entry_row_wrapper.tsx

```diff
@@ -31,6 +31,7 @@ export const LogEntryRowWrapper = euiStyled.div.attrs(() => ({
 export interface LogEntryRowWrapperProps {
   scale: TextScale;
   isHighlighted?: boolean;
+  alt: string;
```

Comment:
The `alt` won't be needed here to fix the originating issue. We're showing text on the page which is a great start. The trick is making screen readers understand that text is structured like a table, so they know how to parse individual rows and cells.